Formula/qpdf.rb

```diff
@@ -20,15 +19,17 @@ class Qpdf < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "72c36f09952ffc50cbde983d248609d7c3be695103348ebf1cdafc84b3d8a608"
   end
 
+  depends_on "cmake" => :build
   depends_on "jpeg-turbo"
   depends_on "openssl@1.1"
 
   uses_from_macos "zlib"
 
   def install
-    system "./configure", "--prefix=#{prefix}"
-    system "make"
-    system "make", "install"
+    mkdir "build" do
+      system "cmake", "-DUSE_IMPLICIT_CRYPTO=0", "-DREQUIRE_CRYPTO_OPENSSL=1", "..", *std_cmake_args
+      system "make", "install"
+    end
```

Comment:
Let's use the template provided by `brew create --cmake`.
```suggestion
    system "cmake", "-S", ".", "-B", "build",
                    "-DUSE_IMPLICIT_CRYPTO=0",
                    "-DREQUIRE_CRYPTO_OPENSSL=1",
                    *std_cmake_args
    system "cmake", "--build, "build"
    system "cmake", "--install", "build"
```